modules/cli/src/main/scala/scala/cli/commands/new/New.scala

```diff
@@ -54,5 +56,22 @@ object New extends ScalaCommand[NewOptions] {
         allowExecve = true
       ).waitFor()
 
+    if (exitCode == 0) {
+      val inputs = options.shared.inputs(
+        remainingArgs.remaining,
+        () => Inputs.default()
+      )(
+        using invokeData
+      ).orExit(logger)
+
+      SetupIde.runSafe(
+        options.shared,
+        inputs,
+        logger,
+        buildOptions,
+        Some(name),
+        Seq.empty
```

Comment:
This currently fails with:
```

[error]  zetashift/tlkit.g8: input file not found
Try viewing the relevant help to see the list of available sub-commands and options.
  scala-cli new --help
  
```
It seems that I'm giving the inputs/options wrong, but I'm not sure why :S